upgrade/network/bican_enable.md

```diff
@@ -0,0 +1,504 @@
+   kubectl get cm -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > manifest.yaml
+   ```
+
+6. (`ncn-m001#`) Backup original NCN Personalization data.
+
+   ```bash
+   cray cfs configurations describe ncn-personalization --format json | jq 'del(.lastUpdated) | del(.name)' > ncn-personalization.json
+   ```
+
+7. (`ncn-m001#`) Backup original BSS data.
+
+   ```bash
+   curl -s -X GET -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/bss/boot/v1/bootparameters | jq . > bss-bootparameters.json
+   ```
+
+8. (`ncn-m001#`) Backup system databases (optional).
+
+   ```bash
+   TODO: DO WE NEED THIS?
+   ```
+
+## Update Phase
+
+### Update SLS
+
+1. (`ncn-m001#`) Move to the update directory.
+
+   ```bash
+   cd ${UPDATEDIR}
+   ```
+
+2. (`ncn-m001#`) Set the directory location for the `SLS` `CHN` script
+
+   ```bash
+   [[ -f /usr/share/doc/csm/upgrade/scripts/sls/sls_can_to_chn.py ]] &&
+      export SLS_CHN_DIR=/usr/share/doc/csm/upgrade/scripts/sls || 
+      echo "STOP:  Manual intervention is required.
+      1. Ensure the [CSM 1.3 Documentation](../../update_product_stream/README.md#check-for-latest-documentation) is installed.
+      2. export SLS_CHN_DIR=/usr/share/doc/csm/upgrade/scripts/sls"
+   ```
+
+3. (`ncn-m001#`) Add `CHN` to `SLS` data.
+
+   ```bash
+   ${SLS_CHN_DIR}/sls_can_to_chn.py --sls-input-file ${BACKUPDIR}/sls_input_file.json
+      --customer-highspeed-network <CHN VLAN> <CHN IPv4 Subnet>
+      --number-of-chn-edge-switches <number of edge switches>
+      --sls-output-file ${UPDATEDIR}/sls_file_with_chn.json
+   ```
+
+   where:
+
+      - `<CHN VLAN>` is the "stub" `VLAN` for the `CHN`.  This is currently used only on the edge switches in access mode, not a trunk through the high speed network.
+      - `<CHN IPv4 Subnet>` is the pre-requisite site-routable IPv4 subnet for the `CHN`
+      - `<number of edge switches>` is typically 2 Arista or Aruba switches, but some pre-production systems have 1
+
+4. (`ncn-m001#`) Upload data to `SLS`.
+
+   ```bash
+   curl --fail -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw-service-nmn.local/apis/sls/v1/loadstate' -F "sls_dump=@${UPDATEDIR}/sls_file_with_chn.json"
+   ```
+
+### Update customizations
+
+1. (`ncn-m001#`) Move to the update directory.
+
+   ```bash
+   cd ${UPDATEDIR}
+   ```
+
+2. (`ncn-m001#`) Set the directory location for the customizations script to add `CHN`.
+
+   ```bash
+   [[ -f /usr/share/doc/csm/upgrade/scripts/upgrade/util/update-customizations.sh ]] &&
+      export CUSTOMIZATIONS_SCRIPT_DIR=/usr/share/doc/csm/upgrade/scripts/upgrade/util || 
+      echo "STOP:  Requires manual steps"
+   grep NETWORKSJSON ${CUSTOMIZATIONS_SCRIPT_DIR}/update-customizations.sh >/dev/null 2>&1 ||
+      echo "STOP:  This does not to update networking.  Manual steps required.
+      1. Ensure the [CSM 1.3 Documentation](../../update_product_stream/README.md#check-for-latest-documentation) is installed.
```

Comment:
I think what we want to say here is

* Be really sure you have the latest 1.3 docs because the scripts we're about to call are shipped with those docs
* Set this env var which points to the sls scripts included with the docs
* Now proceed to do these things that run those scripts

Reply:
I've put docs requirements and links up in prerequisites and removed this and other shenanigans like this.